k8s/bank/templates/deployment.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "bank.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "bank.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "bank.labels" . | nindent 6 }}
+  strategy:
+    type: {{ .Values.deployment.strategy.type }}
+    rollingUpdate:
+      maxSurge: {{ .Values.deployment.strategy.maxSurge }}
+  template:
+    metadata:
+      labels:
+        {{- include "bank.labels" . | nindent 8 }}
+    spec:
+      serviceAccountName: {{ .Values.serviceAccount.name }}
+      containers:
+        - name: {{ .Values.deployment.template.specNginx.name }}
+          image: {{ .Values.deployment.template.specNginx.image }}
+          ports:
+            - containerPort: {{ .Values.deployment.template.specNginx.ports.containerPort }}
+              name: {{ .Values.deployment.template.specNginx.ports.name }}
+              protocol: {{ .Values.deployment.template.specNginx.ports.protocol }}
+          volumeMounts:
+            - name: {{ .Values.volumeMounts.name }}
+              mountPath: {{ .Values.volumeMounts.mountPath }}
+        - name: {{ include "bank.fullname" . }}
+          image: {{ .Values.image.repository }}:{{ .Values.image.tag | default "latest" }}
+          env:
+          - name: PG_DATABASE
```

Comment:
lets not hardcode this values. This values must be settable

Reply:
All hardcode values ​​are described in value file